src/coreclr/src/vm/pgo.cpp

```diff
@@ -8,11 +8,74 @@
 #ifdef FEATURE_PGO
 
 ICorJitInfo::BlockCounts* PgoManager::s_PgoData;
-unsigned                  PgoManager::s_PgoIndex;
+unsigned volatile         PgoManager::s_PgoIndex;
 const char* const         PgoManager::s_FileHeaderString  = "*** START PGO Data, max index = %u ***\n";
 const char* const         PgoManager::s_FileTrailerString = "*** END PGO Data ***\n";
 const char* const         PgoManager::s_MethodHeaderString = "@@@ token 0x%08X hash 0x%08X ilSize 0x%08X records 0x%08X index %u\n";
 const char* const         PgoManager::s_RecordString = "ilOffs %u count %u\n";
+const char* const         PgoManager::s_ClassProfileHeader = "classProfile iloffs %u samples %u entries %u totalCount %u %s\n";
+const char* const         PgoManager::s_ClassProfileEntry = "class %p (%s) count %u\n";
+
+struct HistogramEntry
+{
+    CORINFO_CLASS_HANDLE m_mt;
+    unsigned             m_count;
```

Comment:
Document the meanings of these fields:

m_count is the number of occurrences of m_mt in ClassTable[]